blocks/hero-carousel/hero-carousel.css

```diff
@@ -18,11 +18,12 @@
   background-color: purple;
 }
 
-/* set emphasis styles */
+/* set emphasis styles 
 .block.hero-carousel .hero-slide.background-green em {
   color: pink;
 }
 
 .block.hero-carousel .hero-slide.background-purple em {
   color: green;
-}
\ No newline at end of file
+}
+  */
```

Comment:
since hero-slide is no longer used based on` component-definition.json`, maybe we can remove this piece altogether?

Reply:
removed hero-carousel .js and .css